core/iwasm/aot/aot_runtime.c

```diff
@@ -1774,7 +1774,7 @@ aot_module_malloc(AOTModuleInstance *module_inst, uint32 size,
             aot_set_exception(module_inst, "app heap corrupted");
         }
         else {
-            aot_set_exception(module_inst, "out of memory");
+            LOG_ERROR("alloc %d bytes memory failed", size);
```

Comment:
Had better change to `LOG_WARNING("warning: allocate %d bytes memory failed", size)`? LOG_ERROR normally means some error happens.